heroic/hc_felhounds_of_sargeras.simc

```diff
@@ -1,11 +1,22 @@
 # movement
 # consuming sphere happens about every minute for about 15 seconds
-raid_events+=/movement,cooldown=60,duration=15,first=60
+raid_events+=/movement,cooldown=65,duration=1.5,first=60,players_only=1
+raid_events+=/movement,cooldown=65,duration=1.5,first=63,players_only=1
+raid_events+=/movement,cooldown=65,duration=1.5,first=66,players_only=1
+raid_events+=/movement,cooldown=65,duration=1.5,first=69,players_only=1
+raid_events+=/movement,cooldown=65,duration=1.5,first=72,players_only=1
```

Comment:
TBH i think we leave these out that are players_only=1 just because if you are doing a solo sim or comparison sim it will skew the data.

Reply:
Players only means that something like Mindbender won't be moving as well. Did you have something else in mind?